doc/userguide/rules/vlan-id.rst

```diff
@@ -0,0 +1,21 @@
+Vlan.id Keyword
+==============
+
+Suricata has a ``vlan.id`` keyword that can be used in signatures to identify
+and filter network packets based on Virtual Local Area Network IDs.
+
+Syntax::
+
+ vlan.id: id[,layer];
```

Comment:
What is the default behavior when we do not specify a layer ?

Reply:
It behaves just like specifying 'any',  it matches any of the VLAN IDs.

Reply:
Then it's good to have that added to the docs :)